app/renderer/js/preload.js

```diff
@@ -7,6 +7,8 @@ const ConfigUtil = require(__dirname + '/utils/config-util.js');
 const LinkUtil = require(__dirname + '/utils/link-util.js');
 const params = require(__dirname + '/utils/params-util.js');
 
+const fallbackLocales = require(__dirname + '/../../resources/spellcheck/fallback.json');
```

Comment:
Hmm, so why do you call this fallback? I feel like there could be a better name for this

Reply:
That's the same language as used by the spellchecker module. I figured that we should use the same thing. 
We can call it `normalizedLocales` if that sounds better to you.

Reply:
`normalizedLocales` sounds better.